static/js/addons/cl.utils.js

```diff
@@ -49,8 +49,12 @@ var Cl = window.Cl || {};
         },
 
         // INFO: simple mobile and tablet filter that can be extended
-        mobile: function () { return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i)); },
-        tablet: function () { return (navigator.userAgent.match(/iPad/i)); }
+        mobile: function () {
+            return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i));
```

Comment:
@intellectronica this is intended to be very high-level and also more of an example. Including an additional library has to be decided on a project-level